anago

```diff
@@ -279,6 +282,15 @@ rev_version_base () {
   logecho -n "Formatting $version_file: "
   logrun -s gofmt -s -w $version_file
   logrun git add $version_file
+
+  # Run select update- scripts required for $version_file changes
```

Comment:
At this point, I think only `update-staging-client-go` will produce changes. The `openapi-spec` scripts will only notice the version was bumped after a tag for it exists in your local git repo.

There's a chicken-and-egg problem that prevents us from including those changes in the commit that gets tagged, unless you fake it by creating a dummy tag and then deleting it. In the long term, those scripts should perhaps be looking at the version file rather than the set of tags that exist.